src/index.js

```diff
@@ -65,6 +66,8 @@ export { CheckableContinuousFlyout };
 export { ScratchVariables };
 
 export function inject(container, options) {
+  options["renderer"] = "scratch";
```

Comment:
Can this go inside of the `Object.assign` call?

Reply:
Done!